Binary files tmp/xml-HTB-1.3/src/main.o and work/main.d/xml-HTB-1.3/src/main.o differ
diff -x '*~' -x '.*' --speed-large-files --minimal -Nru tmp/xml-HTB-1.3/src/Makefile work/main.d/xml-HTB-1.3/src/Makefile
--- tmp/xml-HTB-1.3/src/Makefile	2004-05-12 05:13:12.000000000 -0300
+++ xml-HTB-1.3/src/Makefile	2008-09-28 16:28:40.000000000 -0300
@@ -4,8 +4,8 @@
 
 CC = gcc
 
-XMLLIBS = `xml2-config --libs`
-XMLCFLAGS = `xml2-config --cflags`
+XMLLIBS = $(shell $(DESTDIR)$(bindir)/xml2-config --libs)
+XMLCFLAGS = $(shell $(DESTDIR)$(bindir)/xml2-config --cflags)
 
 OBJS = main.o parse-doc.o stack.o
 PROG = xml-HTB
Binary files tmp/xml-HTB-1.3/src/parse-doc.o and work/main.d/xml-HTB-1.3/src/parse-doc.o differ
Binary files tmp/xml-HTB-1.3/src/stack.o and work/main.d/xml-HTB-1.3/src/stack.o differ
Binary files tmp/xml-HTB-1.3/src/xml-HTB and work/main.d/xml-HTB-1.3/src/xml-HTB differ
